src/AppNavigator.tsx

```diff
@@ -215,17 +214,9 @@ const useOsAuthDisabled = () => {
   return !canEnableOsAuth && authMethod?.OS === true
 }
 
-const useCheckOsAuthDisabled = (osAuthDisabled: boolean, authMethod?: AuthMethodState) => {
+const useCheckOsAuthDisabled = (osAuthDisabled: boolean) => {
   const strings = useStrings()
   const {logout} = useAuth()
-  const {refetch} = useCanEnableAuthOs()
```

Comment:
are you getting rid of this or moving it?

Reply:
deleted